packages/patternfly-4/react-core/src/components/Modal/Modal.js

```diff
@@ -106,7 +104,7 @@ class Modal extends React.Component {
       this.container = document.createElement('div');
     }
 
-    return ReactDOM.createPortal(<ModalContent {...props} title={this.props.title} id={this.id} ariaDescribedById={this.props.ariaDescribedById}/>, this.container);
+    return ReactDOM.createPortal(<ModalContent {...this.props} title={this.props.title} id={this.id} />, this.container);
```

Comment:
There's definitely something weird happening with `aria-describedby`.

I'd suggest looking at how this prop is used in the [current example labeled **No Header Modal**](https://patternfly-react.surge.sh/patternfly-4/components/modal). The idea is that if `aria-describedby` is not defined, it will default to referencing the `id` defined for `.pf-c-modal-box__body`.

But if the consumer needs to control what contents are used for the description, the consumer can specify an `id` for `aria-describedby` and define that `id` on some chunk of contents anywhere inside `pf-c-modal-box__body`. This is what's being illustrated with  **No Header Modal**.

Let me know if you need more details on this one.

Reply:
This was also a matter of updating the docs. Fixed in latest commit.